flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -2846,17 +2846,17 @@
 
                 <Component ID="Overhead_Back_Panel">
                     <!-- "BLUE PUMP OVRD" -->
-
+                    <!-- This name is wrong, but it refers to the node we want -->
```

Comment:
**question**: What name does this comment refer to?

Reply:
I think previour author refered to the node id naming PUSH_OVHD_HYD_BLUEPUMP, which is easily mixed with the other blue pump pushbutton on hyd panel, but I'm not sure if it's relevant to say it's wrong if there is not particular naming rules for those....   I won't be against removing this :)

Reply:
I'd remove it. There's also a comment indicating which button it is so it should be enough 😃.